Documentation/Installation/Index.rst

```diff
@@ -42,6 +42,9 @@ For all available tags, please check the `GitHub packages page`_.
 Once you have pulled the image, you can run the image to render your project's
 documentation.
 
+Note that we will use `:stable` at a later point to always provide stable
```

Comment:
What about `.. note::`` ;-)

Reply:
I still can't write RST format organically :D Damn, you noticed!

(Will add it now that you mentioned the syntax *g*)